### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by threetriangles**

### CODE REVIEW

File: exercises/pokemon_server.livemd [hunk 1]
```diff
@@ -117,7 +117,7 @@ Pokemon health should not fall below 0.
 
 ```elixir
 {:ok, charmander} = Pokemon.new(%{name: "charmander", type: "fire", damage: 110})
-{:ok, bulbasaur} = Pokemon.new(%{name: "charmander", type: "fire", health: 100})
+{:ok, bulbasaur} = Pokemon.new(%{name: "bulbasaur", type: "fire", health: 100})
```
Comment (zorn) [inline exercises/pokemon_server.livemd:120]:
```suggestion
{:ok, bulbasaur} = Pokemon.new(%{name: "bulbasaur", type: "water", health: 100})
```

Comment (zorn) [reply exercises/pokemon_server.livemd:120]: I think this should be water.

File: exercises/pokemon_server.livemd [hunk 2]
```diff
@@ -117,7 +117,7 @@ Pokemon health should not fall below 0.
 
 ```elixir
 {:ok, charmander} = Pokemon.new(%{name: "charmander", type: "fire", damage: 110})
```
Comment (zorn) [inline exercises/pokemon_server.livemd:119]: I think there is an inconsistency in that I don't see `damage` mentioned at the top of the exercise. I do see `attack`. Maybe this should be fixed as well?

I also see a curious line where it appears the exercise wants us to make a `Pokemon.attack/2` function, but I see inconsistent return values, sometimes being the first monster and sometimes the second. Not being familiar with the work around this exercise I might be mistaken but just sharing as I read this over when seeing the PR.

Comment (BrooklinJazz) [reply exercises/pokemon_server.livemd:119]: Yup, they should both be `attack`. That's a mistake on my part.

File: exercises/pokemon_server.livemd (hunk 1 above)
Comment (BrooklinJazz) [reply exercises/pokemon_server.livemd:120]: Yup!

**Review by BrooklinJazz — APPROVED** [review]
There are a couple of touch-ups I'm going to make to this exercise to address comments on this PR. Great catches.

**Pull request merged**